src/lib.rs

```diff
@@ -4,7 +4,7 @@
 
 #![feature(specialization)]
 #![feature(non_exhaustive)]
-#![feature(box_syntax)]
+#![feature(box_syntax, trait_alias)]
```

Comment:
Put the `trait_alias` feature on an own line

Reply:
Addressed in b4919d9